app/src/widgets/MessageList/ui/MessageList.tsx

```diff
@@ -8,13 +8,13 @@ import {
 } from 'react';
 import { useTranslation } from 'react-i18next';
 import cn from 'classnames';
+import { isUndefined } from 'lodash';
```

Comment:
```suggestion
import isUndefined from 'lodash/Undefined';
```

или как-то так будет